esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -365,28 +371,13 @@ then
     read THROWAWAY
 fi
 
-# Mailman setup
-# To reset: remove list manually
-if [[ "$MODE_MAILMAN" || "$MODE_ALL" ]]
-then
-    newlist -q $SITENAME-websupport $ADMINEMAIL $MAILMAN_LIST_PASSWORD
-    add_members -r - granite-websupport <<EOF
-$ADMINEMAIL
-EOF
-    echo "Created Mailman list $SITENAME-websupport with initial member $ADMINEMAIL."
-    
-    echo "Support list has been created.  Please check them by looking over the"
-    echo -n "output above, then press enter to continue or Ctrl-C to quit."
-    read THROWAWAY
-fi
-
 # Apache setup
 # To reset: remove appropriate section from Apache config
 if [[ "$MODE_APACHE" || "$MODE_ALL" ]]
 then
     cat >>$APACHE_CONF_FILE <<EOF
 #   $INSTITUTION $GROUPNAME (automatically generated)
-WSGIDaemonProcess $SITENAME processes=1 threads=1 maximum-requests=1000
+WSGIDaemonProcess $SITENAME processes=2 threads=1 maximum-requests=1000
```

Comment:
My interpretation of the documentation and Internet commentary is that the number of processes you specify using ``processes`` is a constant, although the number of threads is allowed to vary up to the limit you specify. (It looks like ``inactivity-timeout`` is for restarting processes, not shutting them down.)  I can't tell what the MaxClients setting would do besides starving the WSGI processes of requests, given that we have 1 thread per process (which I'm guessing is still required by our code... though it would be interesting to test).  So I don't think we can do things much differently than the current configuration.  

Let's just allocate 4+ processes to everyone and put in more memory when we get the chance.  Right now we have a total of 126 WSGI processes allocated, and ps_mem.py reports that Apache2 has a total of 135 processes using 5.6 GiB of memory.